assets/home/components/HomeApp.tsx

```diff
@@ -22,6 +22,7 @@ import {personalizeHome} from 'agenda/actions';
 import {RadioButtonGroup} from 'features/sections/SectionSwitch';
 import {getCurrentUser} from 'company-admin/selectors';
 import {IPersonalizedDashboardsWithData} from 'home/reducers';
+import '@superdesk/common/dist/src/index.css';
```

Comment:
wasn't it already imported from base component meaning it wouldn't need additional imports?

Reply:
it is, but the HomeApp doesn't extend the base, therefor this import wouldn't be available in the homeapp if not explicitly imported as I did

Reply:
What else doesn't extend the base? If this is the only one then fine, but if there are others we should to come up with a way to import it once in the project